htdocs/reports/reports_testhistory.php

```diff
@@ -2079,13 +2106,70 @@ function change_to_bold() {
 	}
 }
 ?>
-<div>
+<div id="signature">
 Signed by:
 <label>
 	<?php $user = get_user_by_id($signature);
 	echo $user->actualName; ?></label>
 </div>
 
+<div id="print_div" class="hidden"></div> <!--Content when the 1 test per page checkmark is checked is moved here-->

```

Comment:
```suggestion
<div id="print_div" class="hidden"></div>
<?php
  # Content when the 1 test per page checkmark is checked is moved here
?>
```

It doesn't really matter much, but my slight preference is to move comments like this into code that won't be rendered to the user.

Reply:
On a separate note, is 1 test "guaranteed" (in whatever sense of the word) to fit on 1 page? (could a single test span 2 pages if there were enough results?)

Reply:
I moved the comment into a PHP block. As far as the 1 test per page being guaranteed I'm not really sure. The only one I could see potentially having a problem right now is the first page since it has some lingering CSS from somewhere. I've did some poking and I couldn't find what exactly is doing what I'm assuming is padding but I'll take a look again. If we did try to ensure 1 page always, if it is a problem currently, I'm not even sure how we would really tackle that outside of doing something along the lines of decreasing font sizes or something like that, but I don't think that would be an optimal solution. Let me know if you think of anything that can be done different.